plasma_framework/test/src/exits/payment/controllers/PaymentProcessStandardExit.test.js

```diff
@@ -79,6 +94,7 @@ contract('PaymentProcessStandardExit', ([_, alice]) => {
             exitTarget,
             amount: web3.utils.toWei('3', 'ether'),
             bondSize: this.startStandardExitBondSize.toString(),
+            bountySize: this.processExitBountySize.toString(),
         });
 
         describe('when paying out bond fails', () => {
```

Comment:
I think we are missing the same test for bounty flow when the transfer fails

Reply:
yess, thanks for catching